core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/SameSiteCookie.java

```diff
@@ -15,8 +15,28 @@
  */
 package org.wso2.carbon.core;
 
+/**
+ * Handle the default values of the sameSite cookie attribute.
+ */
 public enum SameSiteCookie {
-    Lax,
-    Strict,
-    None
+    LAX,
+    STRICT,
+    NONE;
+
+    /**
+     * Handle the sameSite translation from enum to as expected in the sameSite Specification.
+     *
+     * @return Expected sameSite value in the sameSite Specification.
+     */
+    public String toPrintable() {
```

Comment:
Let's use something like this https://stackoverflow.com/questions/17741721/getting-string-value-from-enum-in-java instead of a switch. 

We can use a field like "name" to keep the printable enum name.